docs/generators/php.md

```diff
@@ -21,6 +21,7 @@ These options may be applied as additional-properties (cli) or configOptions (pl
 |allowUnicodeIdentifiers|boolean, toggles whether unicode identifiers are allowed in names or not, default is false| |false|
 |apiPackage|package for generated api classes| |null|
 |artifactVersion|The version to use in the composer package version field. e.g. 1.2.3| |null|
+|booleanFormatForQueryString|Specifies the format for converting boolean to query string. The default is 'int'.(e.g. int param=1 or param=0 string param=true or param=false)|<dl><dt>**int**</dt><dd>int</dd><dt>**string**</dt><dd>string</dd></dl>|int|
```

Comment:
Additional codegen option, really? Are you sure there are many users who need this?
Beside why not to add that option to `modules/openapi-generator/src/main/resources/php/Configuration.mustache` instead? That way you can change bool serialization without regeneration of client SDK.

Reply:
Additional option is better then option to `Configuration.mustache`
the reason -  it is need for API only 
if your API doesn’t handle option=1 only option=true 
no necessary for an user to set up it because it will be common behavior for all requests
and in our case all our API requests handle string format 

no, I think it is necessary for a small number of users (one more reason for using Additional option)
СС @ybelenko

Reply:
@wing328 As far as I remember you also against adding dozens of options to codegen. In current situation I think `Configuration` php class is better place to put this option.

@ksvirkou-hubspot Your suggested codegen option changes one single line of code, I understand that it's convinient to you, but client SDK users always ask to consider their use case as one and only important. 😅

Reply:
Ok
it was only thought 
waiting decision and will do it